pkg/base/addr_validation.go

```diff
@@ -274,23 +277,29 @@ func (cfg *Config) CheckCertificateAddrs(ctx context.Context) {
 		// Log the certificate details in any case. This will aid during troubleshooting.
 		log.Infof(ctx, "server certificate addresses: %s", addrInfo)
 
-		// Verify the compatibility. This requires that ValidateAddr() has
+		// Verify the compatibility. This requires that ValidateAddrs() has
 		// been called already.
-		var buf strings.Builder
-		notifyFunc := func(msg, host string) {
-			if buf.Len() > 0 {
-				buf.WriteString(" and ")
+		host, _, _ := net.SplitHostPort(cfg.AdvertiseAddr)
```

Comment:
Is there a reason you're ignoring the returned error? Have there been valid addresses we failed on?

Reply:
I'm assuming here `ValidateAddrs()` has been called already. If the syntax was not correct that would have failed before. Do you think I should still check here with a `panic("call ValidateAddrs first")`?

Reply:
I'm fine leaving it out, mostly curious.

Reply:
ok added the panic